extensions/-SIPC-/consoles.js

```diff
@@ -6,18 +6,18 @@
         constructor () {}
         getInfo() {
             return {
-                id: 'sipcconsole',
+                id: 'sipcconsoles',
```

Comment:
can't change this

Reply:
all of your other changes seem backwards-compatible so just don't touch the extension id

Reply:
Oh! I wasn't aware that ’s‘ was added here